# HG changeset patch
# User David Vrabel <david.vrabel@citrix.com>
# Date 1400160721 -7200
# Node ID 102d1430db962f37f39510ce1679869ca3d4fafe
# Parent  2fb909141ccf72b55880489e59024f375d7a835c
x86/nmi: remove spurious local_irq_enable from check_nmi_watchdog()

All callers of check_nmi_watchdog() already have local irqs enabled so
remove the unpaired local_irq_enable().

Signed-off-by: David Vrabel <david.vrabel@citrix.com>
Reviewed-by: Andrew Cooper <andrew.cooper3@citrix.com>
Reviewed-by: Tim Deegan <tim@xen.org>

diff -r 2fb909141ccf -r 102d1430db96 xen/arch/x86/nmi.c
--- a/xen/arch/x86/nmi.c
+++ b/xen/arch/x86/nmi.c
@@ -126,7 +126,6 @@ int __init check_nmi_watchdog (void)
 
     for_each_online_cpu ( cpu )
         prev_nmi_count[cpu] = nmi_count(cpu);
-    local_irq_enable();
 
     /* Wait for 10 ticks.  Busy-wait on all CPUs: the LAPIC counter that
      * the NMI watchdog uses only runs while the core's not halted */
